README.md

```diff
@@ -26,33 +27,59 @@ Alternativly, simply add these lines to your /etc/hosts:
 ::1 ci.foo.redhat.com
 ```
 
-## Getting started
+## Getting started - Installation
 
 1. Make sure [nvm](https://github.com/nvm-sh/nvm) is installed
 
-2. First time running the app do: `nvm use` to ensure you have the correct node version installed. If you do not, follow the instructions nvm gives you to install the appropriate version.
+2. First time running the app do: `nvm use` to ensure you have the correct node version installed.
```

Comment:
```suggestion
2. First time running the app do: `nvm use` to ensure you have the correct node version installed.
```
If this is important then no need to say "First time running the app do:". Rather tell the user what version they need and say sonmehting short and to the point, e.g. "Ensure node version X is in use"